src/fable-fsharp/Replacements.fs

```diff
@@ -190,6 +193,8 @@ module Util =
                     |> makeCall com i.range (Fable.PrimitiveType (Fable.Number Int32))
                 Fable.Apply(op, [comp; makeConst 0], Fable.ApplyMeth, i.returnType, i.range)
                 |> Some
+            | Fable.Class None ->  // For enums only ("DeclaredType <enumName> Class null")
+                Fable.Apply(op, args, Fable.ApplyMeth, i.returnType, i.range) |> Some  
```

Comment:
P.s.: I understand why you think it can be dangerous. And I fully agree. But every test passes until now. Could you think a way this could go wrong? If yes, please add a test to it. 

Anyway, This is only to fix the case when the first operand is not wrapped and the second is. If you there's a better way, please tell me :)